fe/fe-core/src/main/java/org/apache/doris/planner/SetOperationNode.java

```diff
@@ -320,30 +321,30 @@ public void init(Analyzer analyzer) {
     }
 
     protected void toThrift(TPlanNode msg, TPlanNodeType nodeType) {
-        Preconditions.checkState( materializedResultExprLists_.size() == children.size());
```

Comment:
still a checkstyle warning here?

Reply:
only care about error in this pr, the purpose of this pr is change some rules' s severity to error and fix all of them.